cirq-core/cirq/study/resolver.py

```diff
@@ -36,6 +36,9 @@
     ParamResolverOrSimilarType, """Something that can be used to turn parameters into values."""
 )
 
+# Used to mark values that are not found in a dict.
+_NotFound = object()
```

Comment:
Shouldn't global variables be named like `_NOT_FOUND` according to the style guide?  Same with RecursionFlag below if you want to clean it up.
go/pystyle#guidelines-derived-from-guidos-recommendations

Reply:
Renamed to `_NOT_FOUND` and `_RECURSION_FLAG`.